.appveyor.yml

```diff
@@ -0,0 +1,217 @@
+# This file is used to configure the AppVeyor CI system, for testing on Windows machines.
+#
+# Code loosely based on https://github.com/ogrisel/python-appveyor-demo
+# 
+# To test with AppVeyor:
+#    Register on appveyor.com with your GitHub account.
+#    Create a new appveyor project, using the GitHub details.
+#    Ideally, configure notifications to post back to GitHub. (Untested)
+
+environment:
+    matrix:
+        # Version Restrictions
+        #    Enaml supports Python 2.7 and also 3.4 onwards.
+        #    PyQt4 only available for Python <3.6.
+        #    PyQt5 only available for Python >=3.5.
+        #    PySide1 only available for 2.6 <= Python <=3.4. Wheels only for 32-bit.
+        
+        # Strategy:
+        #    Test each 64-bit Python version, but only the latest known patch.
+        #    Test the oldest and the latest 32-bit Python versions release. 
+        #    Test with PyQt4 where available, otherwise PyQT5 in each version.
+        #    Test Pyside1 in latest Python version it runs on.
+        #    Test Pyside2 in latest Python version available.
+        #    Test QScintilla on latest Python version available.
+        #    Only test single toolkit installs (e.g. not QT4 and QT5 on same install)
+        
+        # VARIABLES
+        #    NAME: just a comment for distinguishing builds in Appveyor.
+        #    PYQT_VERSION: Determines PyQt versions installation
+        #    PY_QT: Determines Pyside version installation (if present), and is used by 
+        #           Enaml.
+        #    PYTEST_QT_API: In Python 3.4, pytest-qt needs a hint about the right interface.
+        #    QSCINTILLA: Set to "Yes" to install it.
+        
+
+        # 64-bit versions, PyQT, latest to oldest.
+        
+        - NAME: "Py 3.6"
+          PYTHON: "C:\\Python36-x64"
+          PYTHON_VERSION: "3.6.3"
+          PYTHON_ARCH: "64"
+          MINICONDA: C:\Miniconda36-x64
+          PYQT_VERSION: "5"
+
+        - NAME: "Py 3.5"
+          PYTHON: "C:\\Python35-x64"
+          PYTHON_VERSION: "3.5.5"
+          PYTHON_ARCH: "64"
+          MINICONDA: C:\Miniconda35-x64
+          PYQT_VERSION: "5"
+
+# Does not pass tests!
+#        - NAME: "Py 3.4 PyQt 4 v2"
+#          PYTHON: "C:\\Python34-x64"
+#          PYTHON_VERSION: "3.4.8"
+#          PYTHON_ARCH: "64"
+#          MINICONDA: C:\Miniconda34-x64
+#          PYQT_VERSION: "4"
+#          PYTEST_QT_API: "pyqt4v2"
+#
+#        - NAME: "Py 3.4 PyQt 4 v1"
```

Comment:
This probably breaks because of qtpy and it is not meant to be supported.

Reply:
I am a little unclear on what it is that isn't supported. 

Travis has a "PYTHON=3.4 QT_VERSION=4" build working on Linux, but I have been unable to get it to work on Windows. In fact, I haven't got any combination (PyQt4, PyQt5, Pyside, Pyside2) to work on Windows with Python 3.4 yet. I haven't ruled out user error though.

If you want to drop Python 3.4 support, that's fine by me. Just seeking clarification.